appveyor.yml

```diff
@@ -23,15 +24,17 @@ init:
       { $Env:APPVEYOR_SKIP_FINALIZE_ON_EXIT = 'true'; Exit-AppVeyorBuild }
 
 install:
-  # Use Python 3.7 by default
-  - "SET PATH=C:\\Python37;C:\\Python37\\Scripts;%PATH%"
+  # Prepare PATH
+  - SET PATH=C:\\Python37;C:\\Python37\\Scripts;C:\\tools\nginx-1.17.1;%PATH%
```

Comment:
No, it will be installed by Chocolatey once nginx plugin is working on Windows. This is a residue of previous attempts to run it right now, but it is to be removed for now.